### PR TITLE
Package ocaml-varint.1.0

### PR DESCRIPTION
**Pull request by XnuKernPoll**
### `ocaml-varint.1.0`

A simple VarInt implementation.

---
* Homepage: https://github.com/XnuKernPoll/ocaml-varint
* Source repo: http://github.com/XnuKernPoll/ocaml-varint.git
* Bug tracker: https://github.com/XnuKernPoll/ocaml-varint/issues

---
### opam-lint failures
- **WARNING** 97 long description unspecified

---

:camel: Pull-request generated by opam-publish v0.3.5

### CODE REVIEW

File: packages/ocaml-varint/ocaml-varint.1.0/descr [hunk 1]
```diff
@@ -0,0 +1 @@
+A simple VarInt implementation.
```
Comment (pmetzger) [inline packages/ocaml-varint/ocaml-varint.1.0/descr]: I note there is no \n at the end of this line.

**Comment by pmetzger** [comment]
Just an aside: when I see interesting package names go by, I often try to look at the package and see if it is interesting to me. The entire documentation (including in your README and your mli files) for this package is

> A simple VarInt implementation.

which makes it hard for potential users to figure out that they might want to use your code.

**Comment by hannesm** [comment]
This may be not well documented (it is in https://github.com/ocaml/opam-repository/wiki/PR-checks), in opam-repository we try to avoid `ocaml-` prefixes and `-ocaml` suffixes for package names.  Would you be so kind to rename your package to `varint` (and I agree with @pmetzger that varint should be a bit better documented).  Thanks for your contribution!

**Comment by XnuKernPoll** [comment]
Alright, I'll fix the package descr, change the name, and add a tutorial in the read me, and add some comments in the mli files.

**Comment by XnuKernPoll** [comment]
I added the changes you guys gave me, fixed the api, and added Length Prefixing
https://github.com/ocaml/opam-repository/pull/12377

Out of curiosity do append, concat, or sub copy, because I know that bigarray can use layout transformation to avoid copies.

**Pull request closed without merging**